third_party/externals.bzl

```diff
@@ -143,6 +144,14 @@ package(default_visibility = ["//visibility:public"])
         strip_prefix = "bazel-compilation-database-0ae6349c52700f060c9a87c5ed2b04b75f94a26f",
     )
 
+    new_git_repository(
+        name = "rubyfmt",
+        remote = "https://github.com/penelopezone/rubyfmt",
+        commit = "f64f80ce80a5027106f689e76e6400abaa87d90f",
+        init_submodules = True,
+        build_file = "//third_party/rubyfmt:rubyfmt.BUILD",
+    )
```

Comment:
Would it be possible for you to use an http_archive for this? Stripe's internal CI places limits on which external services we can call out to from behind a proxy, and GitHub is excluded (it's why you see all those artifactory fallback URLs in this file)

Reply:
I can provide this as a http archive.

Reply:
done.